source/module_basis/module_pw/test/Makefile

```diff
@@ -94,7 +94,7 @@ endif
 ##==========================
 ## GTEST 
 ##==========================
-GTESTOPTS = -I${GTEST_DIR}/include -L${GTEST_DIR}/lib -lgtest -lpthread
+GTESTOPTS = -I/usr/local/gtest/include -L/home/ubuntu/desktop/github/googletest/lib -lgtest -lpthread
```

Comment:
Do not change it

Reply:
The Makefile has been changed,it can be complied in the docker file.The is the result.
![图片](https://github.com/user-attachments/assets/744a86d4-c7c6-48b4-a4cd-ed5f026263d7)